draft-ietf-httpbis-digest-headers.md

```diff
@@ -412,67 +411,110 @@ All digest-algorithm token values are case-insensitive
 but lower case is preferred;
 digest-algorithm token values MUST be compared in a case-insensitive fashion.
 
-The Internet Assigned Numbers Authority (IANA) maintains a registry for
+The "HTTP Digest Algorithm Values Registry",
+maintained by IANA at <https://www.iana.org/assignments/http-dig-alg/> registers
 digest-algorithm values.
-The registry is initialized with the tokens listed below.
-
-Deprecated digest algorithms MUST NOT be used:
-
-- "md5", see [CMU-836068] and {{?NO-MD5=RFC6151}};
-- "sha", see [IACR-2020-014] and {{?NO-SHA1=RFC6194}}.
-
-See the references above for further information.
+Registrations MUST include the following fields:
+
+ - Digest algorithm: the token value.
+     The registry can be used to reserve token values
+ - Status: the status of the algorithm.
+     Use "standard" for standardized algorithms without known problems;
+     "experimental", "obsoleted", or some other appropriate value
+     - e.g. according to the type and status of the primary document
+     in which the algorithm is defined;
+     "deprecated" when the algorithm is insecure or otherwise undesirable;
+     "reserved" when Digest algorithm references a reserved token value
+ - Description: the description of the digest-algorithm and its encoding
+ - Reference: a set of pointers to the primary documents defining the digest-algorithm
+
+Values to be added to this registry require IETF Review (see Section 4.8 of {{!RFC8126}}).
+The associated encoding for new digest-algorithms MUST either
+be represented as a quoted string
+or MUST NOT include ";" or "," in the character sets used for the encoding.
+
+Deprecated digest algorithms MUST NOT be used.
 
+The registry is initialized with the tokens listed below.
 
   {: vspace="0"}
-  sha-256
-  : * Description: The SHA-256 algorithm [RFC6234].  The output of
+  sha-512
+  : * Digest Algorithm: sha-512
```

Comment:
The repetition of the algorithm in each entry is a bit jarring. Maybe we should just present all algorithms in a table form.

Reply:
Tables are an editorial hell... I'd rather strip the `Digest Algorithm:` line ;)

Reply:
fair point. I can live with duplicating